etc/inc/disable-common.inc

```diff
@@ -512,71 +513,73 @@ blacklist ${PATH}/crontab
 blacklist ${PATH}/doas
 blacklist ${PATH}/evtest
 blacklist ${PATH}/expiry
-blacklist ${PATH}/fusermount
+blacklist ${PATH}/fping
+blacklist ${PATH}/fping6
+blacklist ${PATH}/fusermount*
```

Comment:
Realizing it's convenient to use `globbing` IMO we should make an exception here and use separate entrees for fusermount and fusermount3.

Reply:
Fair enough. My intention wasn't to question blacklisting of SUID binaries, simply nitpicking about the comment/section heading (if one can call it that) :)